Honeybadger/screenboard.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@joshuap Thanks. One last request.  What would a user search on to get those events? I'd like to add that to the documentation tile so they know how to get started.  Otherwise, this looks great and is already in our staging environment.